### PR TITLE
Updated Service Provider Workflow and Guidelines to include the use o…

### PR DESCRIPTION
**Pull request by linathedog**
Updated Service Provider Workflow and Guidelines for registering services using the Federation Registry

### CODE REVIEW

**Comment by gwarf** [comment]
Thanks @linathedog, please address the reports from the checks, you can find information about the formatting rules at https://docs.egi.eu/about/contributing/style/. Prettier may help with the formatting.

**Review by gwarf — CHANGES_REQUESTED** [review]
Please address the reported errors.

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -252,27 +262,29 @@ the authenticated user.
 ### Client registration
 
 Before your service can use the EGI Check-in OIDC Provider for user login, you
-must set up a client at <https://aai-dev.egi.eu/oidc/manage/#admin/clients> in
-order to obtain OAuth 2.0 credentials and register one or more redirect URIs.
-The client configuration should include the general information about your
-service, as described in [General Information](#general-information) section.
+must submit a service registration request using
+[Federation Registry](https:/aai.egi.eu/federation) in order to obtain OAuth 2.0
+credentials and register one or more redirect URIs. The client configuration
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:267]:
```
credentials. The client configuration
```

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -252,27 +262,29 @@ the authenticated user.
...
+should include the general information about your service, as described in
+[General Information](#general-information) section.
 
 #### Obtaining OAuth 2.0 credentials
 
 You need OAuth 2.0 credentials, including a client ID and client secret, to
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:273]:
```
You need OAuth 2.0 credentials, which typically include a client ID and client secret, to
```

File: content/en/providers/check-in/sp/_index.md [hunk 3]
```diff
@@ -58,25 +58,32 @@ and specifically are:
...
 To integrate your Service Provider with the EGI Check-in service, you need to
-[open a GGUS ticket](https://ggus.eu/?mode=ticket_submit) indicating your
-request. The responsible support unit is
+create a registration request using the
+[EGI Federation Registry Portal](https://aai.egi.eu/federation) more details in
+the [Federation Registry Documentation](https://federation.rciam.grnet.gr/docs). Service managment
+in the Federation Registry is achieved by submiting service requests. A service
+request can either request to registrer a new service or reconfigure/deregister
+an existing deployed service. Submitted requests will stay pending until
+reviewed by an Operator. The responsible support unit is
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:67]:
```
[EGI Federation Registry Portal](https://aai.egi.eu/federation). You can also use the Federation Registry portal to request the reconfiguration or deregistration of 
an existing deployed service. Service registration requests typically require approval
by an administrator. Please refer to the [Federation Registry Documentation](https://federation.rciam.grnet.gr/docs) for more information. 

The integration follows a two-step process:
```

**Comment by gwarf** [comment]
Should we re-open this PR?

**Pull request closed without merging**